app/src/main/java/org/fossasia/openevent/general/auth/RefreshTokenService.kt

```diff
@@ -0,0 +1,14 @@
+package org.fossasia.openevent.general.auth
+
+import org.fossasia.openevent.general.BuildConfig
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RefreshTokenService {
+
+    private val retrofit = Retrofit.Builder()
+        .addConverterFactory(GsonConverterFactory.create())
+        .baseUrl(BuildConfig.DEFAULT_BASE_URL)
+        .build()
+    val api: AuthApi = retrofit.create(AuthApi::class.java)
+}
```

Comment:
Make it like other classes with DI, and manually inject

Reply:
Manually inject this class into the TokenAutheticator?

Reply:
Yes

Reply:
@iamareebjamal take a look and i also couldn't figure out a method to logout user in case if the refresh token is expired

Reply:
Delete the tokens

Reply:
Oh yes right 😅